test/controllers/SurveyControllerSpec.scala

```diff
@@ -33,7 +33,20 @@ final class SurveyControllerSpec extends AppSpec {
   private val controller = app.injector.instanceOf[SurveyController]
   private val startJourneyController = app.injector.instanceOf[SsjController]
 
-  "survey should render the survey page if the journey is found " in {
+  "survey default should return OK for default" in {
+    val putInDb = startJourneyController.startJourney()(r.withBody[SsjJourneyRequest](ssjJourneyRequest))
+    val ssjResponse = Json.parse(contentAsString(putInDb)).as[SsjResponse]
+    val fakeRequest = FakeRequest("GET", "/")
+
+    val result = controller.surveyDefault(fakeRequest)
+    println(contentAsString(result))
+    status(result) shouldBe Status.OK
+    contentAsString(result).contains("How was our payment service?") shouldBe true
```

Comment:
if you bring in Jsoup then this could be made more precise like:
```scala
val doc = Jsoup.parse(contentAsString(result))
doc.select("h1.govuk-heading-xl").text shouldBe "How was...blah..."
```
Is the content for the entire page being tested anywhere else? If it's not then it'd be worth testing the whole page perhaps

Reply:
if you need some help with css selectors to pinpoint certain elements on the page then there's a couple of resources online that describe it - this one goes over some basic selectors:
https://www.sitepoint.com/css-selectors/

Reply:
something like what I have now? I wasnt sure from looking at that stuff how to seperate by id though